addon/components/polaris-text-field/resizer.js

```diff
@@ -0,0 +1,132 @@
+import Component from '@ember/component';
+import layout from '../../templates/components/polaris-text-field/resizer';
```

Comment:
Move this below the other `@ember` imports so the Ember imports are grouped together